### ISSUE TITLE
some question in reproduce process

### ISSUE THREAD

**Issue by huaqlili** [YBGou/2024-ICML-TAO#4]
Thank you very much for your excellent work. I have noticed that the results obtained from running the code differ from those reported in the paper and the results provided by the authors. Some samples show poorer restoration performance. Could you please confirm if the hyperparameters and configurations used during evaluation are the same as those in the code? If they are different, could you kindly provide the detailed configuration?

**Comment by huaqlili** [comment]
I also noticed that using different random seeds leads to varying results, which seems to be related to the randomness in the Gaussian sampling of the diffusion model. Could you kindly provide the random seeds used for testing on each dataset?

**Comment by YBGou** [comment]
Thanks for your feedback and for bringing these observations to our attention. We confirm that the hyperparameters and configurations used for the paper are indeed the same as those provided in the code. The results located in the directory `test_samples/{dataset's name}/results` correspond to the last run of the code before uploading it.

The random seed has been preset in the code: `10` for dehazing and low-light enhancement, and `20` for denoising. However, we also observe that, even with the same seed, different runs on the same server yield consistent results, while fluctuations can occur when running on different servers. We recommend adjusting the hyperparameters in the code (Lines 123-125, 160-172, and 229 in the `sample_xxx.py` files) to accommodate the variations. This should help improve the alignment of the results with those in the paper. We hope this helps clarify your concerns, and please feel free to reach out if further assistance is needed.

**Comment by progressforever** [comment]
> I also noticed that using different random seeds leads to varying results, which seems to be related to the randomness in the Gaussian sampling of the diffusion model. Could you kindly provide the random seeds used for testing on each dataset?

Have you reproduced the results in this paper? The results are too influenced by the equipment and random seeds, and my results show significant differences.

**Comment by huaqlili** [comment]
Hello,&nbsp; I mainly run the seeds in the author's paper, and when i use other seeds,&nbsp; the results don't&nbsp; be better.&nbsp; It may heavily rely on the randomness of initial noise sampling, and different pytorch version may influence.&nbsp;



---Original---
From: "Zhidong ***@***.***&gt;
Date: Sun, May 25, 2025 14:26 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [YBGou/2024-ICML-TAO] some question in reproduce process (Issue#4)


progressforever left a comment (YBGou/2024-ICML-TAO#4)
  
I also noticed that using different random seeds leads to varying results, which seems to be related to the randomness in the Gaussian sampling of the diffusion model. Could you kindly provide the random seeds used for testing on each dataset?
  
Have you reproduced the results in this paper? The results are too influenced by the equipment and random seeds, and my results show significant differences.
